imports/plugins/core/ui/client/components/table/sortableTable.js

```diff
@@ -123,7 +126,7 @@ class SortableTable extends Component {
 
     // Add minWidth = undefined to override 100px default set by ReactTable
     const displayColumns = columnMetadata.map((element) => {
-      return _.assignIn({}, element, {
+      return _.extend({}, element, {
```

Comment:
Thanks, I'll change those. I inherited those changes and haven't looked at the in detail